site/pages/what-we-do/header-slice/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styles from './style.css';
+import arrowImg from './arrow.png';
+
+const headerSlice = () => {
+  return (
+    <div className={styles.pageHeaderContainer}>
```

Comment:
perhaps a <section> would be marginally better semantically / for accessibility?

Reply:
Make sense 👍 It also keeps the consistency with the other slices.